backend/app/consts.py

```diff
@@ -62,6 +63,7 @@ class PanelId(str, Enum):
     Collection.SERVICE: PanelId.SERVICES,
     Collection.TRAINING: PanelId.TRAININGS,
     Collection.SOFTWARE: PanelId.SOFTWARE,
+    Collection.BUNDLE: PanelId.BUNDLE,
```

Comment:
It is not the way. We should not send recommendation requests at all for bundles (like for providers or guidelines for example)

Reply:
Not relevant for now